lib/pdk/cli/exec.rb

```diff
@@ -46,9 +50,24 @@ def self.git_bindir
 
       def self.git(*args)
         vendored_bin_path = File.join(git_bindir, 'git')
-        git_path = File.exists?(vendored_bin_path) ? vendored_bin_path : 'git'
-        PDK.logger.debug(_("Using git from the system PATH, instead of '%{vendored_bin_path}'") % { vendored_bin_path: vendored_bin_path})
```

Comment:
note that this was always logging this message even when the vendored bin was being used :)